server/middlewares/authentication.js

```diff
@@ -94,78 +91,6 @@ export default function auth(options?: { required?: boolean } = {}) {
       ctx.state.user = user;
     }
 
-    ctx.signIn = (user: User, team: Team, service, isFirstSignin = false) => {
```

Comment:
Moved to `util` method rather than bundling with authentication middleware, as this meant to get the `signIn` method you also had to have token parsing in the stack.